library/objective-c/EnvoyEngineImpl.m

```diff
@@ -16,11 +16,15 @@ static void ios_on_engine_running(void *context) {
   }
 }
 
-static void ios_on_exit(void *context) {
-  // Currently nothing needs to happen in iOS on exit. Just log.
-  NSLog(@"[Envoy] library is exiting");
+static void ios_on_exit(void *context) { NSLog(@"[Envoy] library is exiting"); }
```

Comment:
Would it make sense to also log this to the logger now?

Reply:
The logger isn't implicitly available here, nor is it even a given that we have a logger. I think this is fine as a placeholder, but we could also simply remove it.